src/components/Login/Login.js

```diff
@@ -62,6 +82,13 @@ const Login = ({ intl, location, dependencies: { dopplerLegacyClient, sessionMan
     return values;
   };
 
+  if(!errorMessage) {
+    const errorText = getForgotErrorMessage(location, intl);
+    if(errorText) {
+      setErrorMessage(errorText)
+    }
```

Comment:
If I am understanding it well, you are using this structure to avoid reading query string for each rendering, in that case, `useMemo` would be a better option: https://es.reactjs.org/docs/hooks-reference.html#usememo

BTW, did you see the great job that React people did, translating all Hooks documentation to Spanish?

Reply:
yes i saw it. i try to apply, thanks

Reply:
Tal vez no es necesario, yo primero intentaría con https://github.com/FromDoppler/doppler-webapp/pull/298#discussion_r302504641

Reply:
fixed